runtime/onert/backend/gpu_cl/operand/ICLTensor.cc

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
I guess `ICLTensor.cc` and `ICLTensor.h` should have same year.
am i right ?
if so, please check the Copyright year.